java/vector/src/main/codegen/templates/BaseWriter.java

```diff
@@ -90,6 +90,7 @@
     void clear();
     void copyReader(FieldReader reader);
     MapWriter rootAsMap();
+    MapWriter rootAsMap(Boolean caseSensitive);
```

Comment:
On a side note this should be boolean not Boolean (but it does not apply since I'm suggesting to remove it)